src/ui/screens/home/controller/ControllerView.qml

```diff
@@ -626,9 +637,22 @@ Item {
         objectName: "ipInfoPanel"
 
         opacity: ipInfoPanel.isOpen ? 1 : 0
+        property int previousOpacity: opacity
         visible: opacity > 0
         z: 1
 
+        onOpacityChanged: {
+            // We only want to record this event when the opacity has _just_ changed.
```

Comment:
Curious: Why _just_ as opacity changes?

Reply:
On the Miro board, this is an event to be recorded whenever the user moves on to the conneciton info screen. In the code that is not really a screen though it's an element that we change the opacity. I will change the comment to make this clearer.

Reply:
```suggestion
            // We only want to record this event when the opacity has _just_ changed.
            // This event is expected to be recorded whenever the moved on to the connection info screen.
            // However that is not a "screen" per se, in the code. When the user moves on to this screen
            // it just changes it's opacity to 1.
```